std/uni.d

```diff
@@ -4888,7 +4888,7 @@ template Utf8Matcher()
                 mixin(dispatch);
         }
 
-        bool match(C)(ref C[] str) const pure @trusted
+        bool match(C)(ref C[] str) scope const pure @trusted
             if (isSomeChar!C)
         {
             return fwdStr!"match"(str);
```

Comment:
How about marking the mixed in fwdStr method as `@safe` and `scope` instead.

Reply:
It's a good idea, but I am only concerned here with adding scope and return annotations.

Reply:
I was mostly asking because it's hard to seen whether fwdStr really does not escape anything, and adding scope to a `@trusted` function is unfortunately unchecked by the compiler.